eldritchShooter2.py

```diff
@@ -168,6 +168,7 @@ def update(self):
                 self.velocityy = 0
 
         ### Will be killed if instances of playerbullets collide with instances
+
```

Comment:
Space here is stupid. Consider removing it.

Reply:
just did